cmd/ndm_daemonset/controller/devicestore.go

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	apis "github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
+	"github.com/openebs/node-disk-manager/pkg/util"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// API to creates the Device resource in etcd
+// This API will be called for each new addDiskEvent
+func (c *Controller) CreateDevice(dvr apis.Device) {
```

Comment:
What does the variable name `dvr` stand-for?

Reply:
Read the rest of the file.. possible "device resource" ? .. initially thought this referred to "device request". May a comment when the first time it is used will be helpful.

Reply:
DeVice Resource. Should I change it ?